Update LICENSE

MattsonCam opened the pull request:
This pr updates the license of the repo.

MattsonCam committed:
Update LICENSE

d33bs reviewed (approved):
Nice job - I'm marking this as approved as I trust your judgement on what's best for this project.

Keep in mind: Creative Commons as an organization recommends against using CC licenses for software ([see here](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software)). Consider using another license - this website provides some good details to this effect: https://choosealicense.com/ .

MattsonCam committed:
Update LICENSE

Updates the license to BSD-3. I vaguely remember mention of the copyright going to CU Anschutz. If this is wrong, please let me know.

MattsonCam commented:
Thanks for looking into this @d33bs, I'm requesting a re-review. I vaguely remember mention of the copyright going to CU Anschutz. If this is wrong though, then please let me know.

d33bs reviewed (commented):
Thanks @MattsonCam - I think we need more information on this before I can definitively say that this is accurate. Mostly I'm concerned that we need to clarify the copyright for the institution but there could be other aspects of this we need to adjust too (like what licenses are permissable through that copyright). I plan to work together with you on finding out more information and follow up here once we understand more.

MattsonCam committed:
Update LICENSE

Changed the copyright in the license

MattsonCam commented:
Just updated the license with the playbook guide. Requesting a re-review @d33bs

d33bs reviewed (approved):
LGTM! Thanks @MattsonCam !

MattsonCam commented:
Thanks @d33bs, merging now!
